api/cpp/esp-idf/slint/CMakeLists.txt

```diff
@@ -30,11 +30,18 @@ set(CMAKE_BUILD_TYPE Release)
 set(BUILD_SHARED_LIBS OFF)
 set(Rust_CARGO_TARGET ${rust_target})
 
+# This variable is set when generating nightly snapshot's CMakeLists.txt
+if (SLINT_NIGHTLY)
+    set(SLINT_FEATURE_EXPERIMENTAL ON)
```

Comment:
I don't think we should do that.
EXPERIMENTAL should still be opt in in the user's CMakeLists.txt

For example, in Rust, even if you use nightly you don't get automatic access to experimental features. You still need to opt in with `#![feature(...)]`

The way I see it, nightly is to test the next release, and the next release doesn't come with experimental enabled)

Reply:
While I disagree, I have a stronger interest in getting this change in, so I'll remove that line then.